projects/packages/jetpack-mu-wpcom/src/features/pages/homepage-notice.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * Homepage notice for the Pages list.
```

Comment:
Maybe we can change the filename `homepage-notice.php` to something more meaningful. With the current naming, it isn't clear what the notice is about.